### DISCUSSION TITLE
Is Rancher Desktop really free?

### DISCUSSION THREAD

**Discussion by ctahok** [rancher-sandbox/rancher-desktop#3969] [Q&A] [answered]
At SUSE website [here ](https://www.suse.com/c/rancher_blog/using-rancher-desktop-for-local-kubernetes-development/#:~:text=Unlike%20Docker%E2%80%99s%20product%2C%20however%2C%20Rancher%20Desktop%20is%20a%20free%20and%20open%20source%20solution%20developed%20by%20SUSE )it says "Unlike Docker’s product, however, Rancher Desktop is a free and open source solution developed by SUSE"
but when installed it shows components developed by Docker Inc. - https://i.imgur.com/atH5TPe.jpeg 
If used in commercial organisation doesn't this require a Docker license? If not why?

**Comment by jandubois** [comment, marked as answer]
The docker components used by Rancher Desktop are the `docker` CLI, the moby container engine, the docker credentials helpers, as well as docker-compose and docker-buildx. All these components are open source projects available under a compatible open source license.

The proprietary product from Docker is "Docker Desktop". It also includes all the above mentioned open source components, but the integration with the host operating system, like file and volume sharing, networking support etc. are all closed source and not available without an additional license from Docker Inc.

These proprietary parts have been replaced with either other open source projects (like Lima on macOS), or have been re-implemented by Rancher Desktop itself.

Rancher Desktop is completely open source and only uses components that are open source as well (with the exception of the host operating system it runs on, of course).

**Reply by ctahok** [reply]
Thanks for clarification this Jan. It would be nice to add your answer to the FAQ.
